markets/agave/commons.ts

```diff
@@ -9,7 +9,7 @@ const MOCK_CHAINLINK_AGGREGATORS_PRICES = {
   HNY: oneEther.multipliedBy('1002.508581').toFixed(),
   STAKE: oneEther.multipliedBy('12.58220000').toFixed(),
   WETH: oneEther.multipliedBy('271.04928').toFixed(),
-  LINK: oneEther.multipliedBy('173.33733').toFixed(),
```

Comment:
Why change this? Did you rerun tests after? The mock values are for tests, they don't need to be accurate.